apps/admin-portal/src/pages/customize.tsx

```diff
@@ -0,0 +1,366 @@
+            <Form.Field>
+                <label>App Background Color</label>
+                <Grid columns={ 12 }>
+                    <Grid.Row>
+                        <Grid.Column>
+                            <Form.Input { ...pageBackground } />
+                        </Grid.Column>
+                    </Grid.Row>
+                </Grid>
+            </Form.Field>
+
+            <Button onClick={ () => handleCompileTheme() }>Update</Button>
+        </Form>
+    );
+};
+
+/**
+ * Customize Page.
+ *
+ * @return {ReactElement}
+ */
+export const CustomizePage = (): ReactElement => {
+
+    const { setAppName, setCopyrightText, setLogo, setProductName, setTheme, state } = useContext(ThemeContext);
+
+    const handleThemeSelect = (theme) => {
+        setTheme(theme);
+    };
+
+    const handleProductNameChange = (e) => {
+        setProductName(e.target.value);
+    };
+
+    const handleLogoChange = (e) => {
+        setLogo(e.target.value);
+    };
+
+    const handleCopyrightTextChange = (e) => {
+        setCopyrightText(e.target.value);
+    };
+
+    const handleAppNameChange = (e) => {
+        setAppName(e.target.value);
+    };
+
+    return (
+        <>
+            <Header className="page-header" as="h1">
+                Application Appearance Settings
+                <Header.Subheader className="sub-header">
+                    Use below fields to change appearance of the application.
+                </Header.Subheader>
+            </Header>
+
+            <Divider hidden />
+
+            <Header as="h3" className="sub-header">Theme</Header>
+            <Divider />
+
+            <Card.Group itemsPerRow={ 9 }>
+                { 
+                    Themes.map((theme, index) => {
+                        return (
+                            <Card key={ index } link onClick={ () => handleThemeSelect(theme) }>
+                                <Image src={ `themes-less/themes/${theme}/preview.jpg` } wrapped ui={ false } />
+                                <Card.Content>
+                                    <Card.Header>{ theme }</Card.Header>
+                                </Card.Content>
+                            </Card>
+                        );
+                    })
+                }
+            </Card.Group>
+
+            <Divider hidden />
+
+            <Header as="h3" className="sub-header">Theme colors</Header>
+            <Divider />
+
+            <CSSForm />
+
+            <Header as="h3" className="sub-header">Customize application identity</Header>
+            <Divider />
+
+            <Form>
+                <Form.Field>
+                    <label>Application Name</label>
+                    <Grid columns={ 4 }>
+                        <Grid.Row>
+                            <Grid.Column>
+                                <Form.Input
+                                    value={ state.appName }
+                                    onChange={ (e) => handleAppNameChange(e) }
```

Comment:
We can use the shorthand here and in other places as well i guess.
```suggestion
                                    onChange={ handleAppNameChange }
```